src/goto-programs/lazy_goto_model.cpp

```diff
@@ -205,9 +205,8 @@ void lazy_goto_modelt::load_all_functions() const
         fn_ids_to_convert.push_back(named_symbol.first);
     }
 
-    // Access all functions to convert them
     for(const irep_idt &symbol_name : fn_ids_to_convert)
-      goto_functions.at(symbol_name);
+      goto_functions.ensure_function_loaded(symbol_name);
```

Comment:
Thanks a lot for this; it would be great if that change could be folded into the code introducing the line the first place so as to avoid redundant comment...